woocommerce/woocommerce-android
# Add Tracks Events for Downloadable Files

Closes #3353. 

This adds tracks events for the downloadable files feature. The events are the same as iOS (https://github.com/woocommerce/woocommerce-ios/pull/3402). 

## Testing

1. Navigate to a product that has downloadable files.
2. Tap Downloadable files
3. Confirm the event `product_detail_view_downloadable_files_tapped` is triggered.
4. Tap on the options menu and choose Download Settings.
5. Modify the Download Limit or expiration. 
6. Tap Done
7. Confirm the event `product_downloadable_files_settings_changed` is triggered.
8. Tap on Add File. Select a file and enter a file URL and file name. 
9. Tap Done. 
10. Confirm the event `products_downloadable_file` with `action` `“added”` is triggered. 
11. Tap on an existing file. 
12. Modify the file URL or name. 
13. Tap on Update.
14. Confirm the event `products_downloadable_file` with `action` `“updated”` is triggered. 
15. Tap on an existing file. 
16. Tap on the options menu → Delete. 
14. Confirm the event `products_downloadable_file` with `action` `“deleted”` is triggered. 

## Submitter Checklist
